metricbeat/module/prometheus/_meta/config.yml

```diff
@@ -7,3 +7,8 @@
   #namespace: example
   #username: "user"
   #password: "secret"
+
+  # This can be used for service account based authorization:
```

Comment:
I think it would be useful to say what it does. Something like 

> It reads the contents of the file once at initialization and then uses the value in an HTTP Authorization header.

Reply:
Makes sense, I updated docs to include this longer explanation